package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sample-locator",
-  "version": "2.1.0",
+  "version": "2.2.1",
```

Comment:
This version doesn't match the version 2.2.2 from package.json. Nevertheless, I would increment the minor version, because this PR add's a new config feature.